rtw_stb_image.h

```diff
@@ -0,0 +1,88 @@
+#ifndef RTW_STB_IMAGE_H
+#define RTW_STB_IMAGE_H
+
+// Disable strict warnings for this header from the Microsoft Visual C++ compiler.
+#ifdef _MSC_VER
+    #pragma warning (push, 0)
+#endif
+
+#define STB_IMAGE_IMPLEMENTATION
+#define STBI_FAILURE_USERMSG
+#include "external/stb_image.h"
+
+#include <cstdlib>
+#include <iostream>
+
+class rtw_image {
+  public:
+    rtw_image() : data(nullptr) {}
+
+    rtw_image(const char* image_filename) {
+        // Loads image data from the specified file. If the RTW_IMAGES environment variable is
+        // defined, looks only in that directory for the image file. If the image was not found,
+        // searches for the specified image file first from the current directory, then in the
+        // images/ subdirectory, then the _parent's_ images/ subdirectory, and then _that_
+        // parent, on so on, for six levels up. If the image was not loaded successfully,
+        // width() and height() will return 0.
+
+        auto filename = std::string(image_filename);
+        auto imagedir = getenv("RTW_IMAGES");
+
+        // Hunt for the image file in some likely locations.
+        if (imagedir && load(std::string(imagedir) + "/" + image_filename)) return;
+        if (load(filename)) return;
```

Comment:
let's not hard code this...
imo best choice would be to let the user specify a path to an image/asset folder

Reply:
nice point